### PR TITLE
Breaks PDF into multiple files

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
# Fixes or Implements [AB#9168](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9168)

* [ ] Enhancement
* [x] Bug
* [ ] Documentation

## Description

Fix/Work around for the problem where PDF report is generate with blank pages.
Adds page number to the footer.

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [x] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

NO

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.  

</br>
</br>
</br>
___
  
## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"?

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

### CODE REVIEW

**Review by sslaws — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/views/reports.vue [hunk 1]
```diff
@@ -134,10 +115,14 @@ export default class ReportsView extends Vue {
     );
     private fullName: string = "";
     private medicationStatementHistory: MedicationStatementHistory[] = [];
+    private medicationStatementHistoryTotal: MedicationStatementHistory[] = [];
     private isLoading: boolean = false;
     private protectiveWordAttempts: number = 0;
     private logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
     private isDataLoaded: boolean = false;
+    private fileMaxRecords: number = 1000;
+    @Ref("report")
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/reports.vue:124]: lets move the anotated ones above the class members

File: Apps/WebClient/src/ClientApp/src/views/reports.vue [hunk 2]
```diff
@@ -134,10 +115,14 @@ export default class ReportsView extends Vue {
     );
     private fullName: string = "";
     private medicationStatementHistory: MedicationStatementHistory[] = [];
+    private medicationStatementHistoryTotal: MedicationStatementHistory[] = [];
     private isLoading: boolean = false;
     private protectiveWordAttempts: number = 0;
     private logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/reports.vue:121]: this will not work. Move it to the mounted method

Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/views/reports.vue:121]: also, do you have the latest from dev? I changed line 113

File: Apps/WebClient/src/ClientApp/src/views/reports.vue [hunk 3]
```diff
@@ -134,10 +115,14 @@ export default class ReportsView extends Vue {
     );
     private fullName: string = "";
     private medicationStatementHistory: MedicationStatementHistory[] = [];
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/reports.vue:117]: nit: medicationStatementHistory -> medicationStatementHistoryPage

**Review by FuriousLlama — CHANGES_REQUESTED** [review]

File: Apps/WebClient/src/ClientApp/src/views/reports.vue [hunk 4]
```diff
@@ -182,8 +223,14 @@ export default class ReportsView extends Vue {
             .then((results) => {
                 if (results.resultStatus == ResultType.Success) {
                     this.protectiveWordAttempts = 0;
-                    this.medicationStatementHistory = results.resourcePayload;
+                    this.medicationStatementHistoryTotal =
+                        results.resourcePayload;
                     this.sortEntries();
+                    console.log(
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/views/reports.vue:229]: should we use our winston logger instead?

**Review by leobcgov — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/views/reports.vue (hunk 1 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/views/reports.vue:115]: inclues action /getter, etc

**Review by FuriousLlama — APPROVED** [review]

**Review by leobcgov — APPROVED** [review]

**Pull request merged**